tensorboard/webapp/metrics/views/card_renderer/utils.ts

```diff
@@ -12,22 +12,25 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+import {ExperimentAlias} from '../../../experiments/types';
 import {Run} from '../../../runs/store/runs_types';
 import {LinkedTime} from '../../types';
 import {PartialSeries, PartitionedSeries} from './scalar_card_types';
 
 export function getDisplayNameForRun(
   runId: string,
   run: Run | null,
-  experimentName: string | null | undefined
+  experimentAlias: ExperimentAlias | null | undefined
 ): string {
-  if (!run && !experimentName) {
+  if (!run && !experimentAlias) {
     return runId;
   }
 
-  const displayName = [experimentName, run ? run.name : '...']
-    .filter(Boolean)
-    .join('/');
+  let displayName = [run ? run.name : '...'].filter(Boolean).join('/');
+
+  if (experimentAlias) {
+    displayName = `[${experimentAlias.aliasNumber}] ${experimentAlias.aliasText}/${displayName}`;
```

Comment:
No action required: to make this format consistent, this is the type of UX that might benefit from the Angular Pipe.

Reply:
I forget, was the new line in the displayName between the number and alias text intentional? That seems counter intuitive.

Reply:
There is no newline there. Github is just wrapping it so it fits on the screen.